src/pages/index.tsx

```diff
@@ -1,17 +1,72 @@
 import type { NextPage } from "next";
+import { Stack } from "@chakra-ui/react";
 import Head from "next/head";
 import site from "../../public/site.json";
 import Page from "../components/page/Page";
 
+import { PlannedMaintenance } from "../components/plannedMaintenance";
+import { OperationalStatus, Service } from "../components/OperationalStatus";
+import { PastIncidents } from "../components/pastIncidents";
+import { ServiceStatus } from "../components/OperationalStatus/OperationalStatus";
+
+import statusJsonData from "../../public/status.json";
```

Comment:
These are compile time imports which is fine for the incidents and planned maintenance. The status data should be fetched in a useEffect hook

Reply:
Updated